### DISCUSSION TITLE
Import graph from neo4j into AGE?

### DISCUSSION THREAD

**Discussion by Elektra-K** [apache/age#118] [Q&A]
Hello, 

Is there any way to export a graph from neo4j (in Cypher or any other format) and import it into AGE?
I've tried with the Cypher file but doesn't seem to work, as there are Cypher commands that are not yet supported in AGE.

Are there any walkarounds?

Thanks

**Comment by ontologiae** [comment]
You could export Neo4j data by using Csv files, import CSV inside PostgreSQL and use them to rebuild the graph
